src/plugin.h

```diff
@@ -42,6 +42,14 @@ class EventHandlerBasePlugin {
 
   _FOR_EACH_EVENT_HANDLER(DEFINE_AND_IMPLEMENT_EVENT_HANDLER_METHOD)
 
+  const uint16_t storageSize() {
```

Comment:
Does this need to be in plugin.h?

Reply:
Kinda, yes, because this is a method that will need to be implemented by plugins, much like the handlers. If you mean the body of it - that can be moved elsewhere and marked up with a `weak` attribute, I guess.

Reply:
right. for merge, documentation will make that clear. also possibly reviewing it in context